arxiv/base/static/js/notification.js

```diff
@@ -0,0 +1,27 @@
+/**
+  * Implements dismissable notifications.
+  *
+  * To use, insert an element (e.g. a button) with the class
+  * `notification-dismiss` that is the direct child of the element that you
```

Comment:
Does the `notification-dismiss` class exist solely to drive the JavaScript for dismissal? If so - does it cause collisions with any other variables to use the Bulma default class `delete` instead? (I recognize it's a common word and may not be usable as such.)

Reply:
Yeah, ``delete`` is pretty broad. I preferred ``notification-dismiss`` for semantic clarity